### PR TITLE
connections and statements can be only closed

### PR DESCRIPTION
**Pull request by c-bik**
removed `*_release` in favor of `*_close`

### CODE REVIEW

**Comment by c-bik** [comment]
> I think we should support both. And leave it up to the user what they want to call.

Yes agree.

But for now (within our scope) there are no need for having `_release`.

**I intend to bring them back when there is a use case where `_close` can't replace `_release`**

File: test/odpi_eunit.erl [hunk 1]
```diff
@@ -1018,7 +1018,7 @@ ref_cursor([Context, Conn]) ->
         <<"">>
     ),
     ?assertEqual(0, dpi:stmt_execute(CreateStmt, [])),
-    ?assertEqual(ok, dpi:stmt_release(CreateStmt)),
+    ?assertEqual(ok, dpi:stmt_close(CreateStmt, <<>>), <<>>),
```
Comment (KarlKeiser) [inline test/odpi_eunit.erl]: I think this is wrong, shouldn't it be
    ?assertEqual(ok, dpi:stmt_close(CreateStmt, <<>>)),
without that second `, <<>>`?

Comment (c-bik) [reply test/odpi_eunit.erl]: Good catch. How is it even compiling!

Comment (c-bik) [reply test/odpi_eunit.erl]: http://erlang.org/doc/man/assert.hrl.html
Third / last param of assert macros are comment

Removing

**Pull request merged**